unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/AvatarEditorHUDController.cs

```diff
@@ -75,9 +80,82 @@ public void SetCatalog(BaseDictionary<string, WearableItem> catalog)
 
     private void LoadUserProfile(UserProfile userProfile)
     {
+        LoadOwnedWereables(userProfile);
         LoadUserProfile(userProfile, false);
     }
 
+    private void LoadOwnedWereables(UserProfile userProfile)
+    {
+        if (!ownedWearablesAlreadyLoaded &&
+            ownedWearablesRemainingRequests > 0 &&
+            !string.IsNullOrEmpty(userProfile.userId))
+        {
```

Comment:
we can invert the if to do an early return so we reduce the nest brackets

Reply:
Good idea! Done in https://github.com/decentraland/explorer/pull/2300/commits/791d4ef062a3530df877ad43dd1929d9e44c6f74. Thanks!